### DISCUSSION TITLE
Could we have a version 1.7.2 before v2 comes out?

### DISCUSSION THREAD

**Discussion by epapbak** [spf13/viper#1130] [General]
As the title says, there are multiple fixes in your master branch, and the latest official tag is 1.7.1, which is almost a year old now. WDYT about it?

**Comment by sagikazarmark** [comment]
We'll release several versions before 2.0. I was planning to release 1.8 after etcd, but I think there are enough changes in master to release a new version.

**Reply by epapbak** [reply]
That's good to know. Thanks!
